### DISCUSSION TITLE
How to build a transaction like this taproot script

### DISCUSSION THREAD

**Discussion by RunlaiDeng** [paulmillr/micro-btc-signer#47] [Q&A]
<img width="1335" alt="image" src="https://github.com/paulmillr/scure-btc-signer/assets/30950645/ced287d1-357e-4127-9071-794a0e0e0778">

**Comment by RunlaiDeng** [comment]
@paulmillr Hi ,a u here?

**Comment by RunlaiDeng** [comment]
Like this 
https://mempool.space/tx/e8e7be7166cec900cfe2568a5cd16ec3ae2fc3d68c31c964a3c1c001df12de1a

**Comment by RunlaiDeng** [comment]
https://github.com/ordinals/ord/blob/f52969fe665697aac5bca4df5948bd0f6abfd037/src/subcommand/wallet/inscribe.rs

Can u check this @paulmillr 

I think its helpful

**Comment by paulmillr** [comment]
You will need to understand it by yourself and post a solution here. It would be useful for the community.

**Reply by RunlaiDeng** [reply]
No , i have no idea about this , im so sad

**Comment by RunlaiDeng** [comment]
Can't you as the author do something, I don't even get an answer using the latest chatgpt4 model with plugins 

@paulmillr

**Reply by paulmillr** [reply]
You want me to code stuff for you, but I am not your free tech support. I have other things to do, really. The library is open-source and free, I am not compensated to work on it full-time.

The issue tracker is for bugs and feature requests. I answer unrelated questions when I have some free time.

**Comment by RunlaiDeng** [comment]
U are right ,Thx for u reply

If you have time, can u check this issue?

**Reply by paulmillr** [reply]
When/if I investigate ordinals/inscriptions, i'd be happy to update docs and post solutions for it.

Thanks for understanding.

**Comment by paulmillr** [comment]
Ordinals got easier with 1.3.0 https://github.com/paulmillr/scure-btc-signer/releases/tag/1.3.0
